C++/syntax_test_c.c

```diff
@@ -10,6 +10,10 @@ int main(){
 }
 
 enum Foo { kFoo, kBar };
+/* <- keyword.declaration */
+/*   ^ entity.name.enum */
+/*         ^ entity.name.enum.constants */
```

Comment:
`entity.name.enum` should be reserved for the enum data type (e.g.: `Foo`).

I'd suggest `entity.name.constant` for `kFoo` and `kBar` as this is what D and Java use.

_C# and Rust scope them `constant.other`, but this should probably be changed._

Reply:
Pulled in the suggested changes for the scope change. Updated the tests to reflect that as well. Thank you for the input! Let me know if there is anything else or something I missed

Reply:
You may probably want to pick the tmPreferences from https://github.com/LinuxDevon/Packages/pull/1 so `entity.name.constant` is properly added to symbol list and index.

Reply:
Just updated the `tmPreference` files as well. I missed the comment earlier.